### DISCUSSION TITLE
Windows package RustDeskServer.Setup.exe file reported as malware

### DISCUSSION THREAD

**Discussion by tomaszs** [rustdesk/rustdesk-server#332] [General]
### Bug Description

The 1.1.9 (https://github.com/rustdesk/rustdesk-server/releases/tag/1.1.9) version ZIP file for Windows contains RustDeskServer.Setup.exe file. The file is reported (https://www.virustotal.com/gui/file/2866c532433d9576beed3389e04e730ca910302f42433b19674773cdbc50354d) by Virus Total to contain malware:

![image](https://github.com/rustdesk/rustdesk/assets/834977/8d511a60-b28c-43e2-99f8-88f59ff43f0d)

### How to Reproduce

Download 1.1.9 Windows ZIP
Scan it with VirusTotal

### Expected Behavior

RustDeck isn't reported as malware

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.1.19

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/834977/663ee257-ff8d-4962-afd4-07c8d0f8f269)

**Comment by CharlieMPK** [comment]
Sorry this is an odd one... Can anyone chime in on this?

**Comment by sebadamus** [comment]
Maybe you can download same nssm version from the official repository and scan it with virus total to compare with the version installed by  RustDeskServer.Setup.exe, if it gives false positive, too.
